lib/ansible/module_utils/basic.py

```diff
@@ -2161,7 +2161,7 @@ def bytes_to_human(self, size):
     pretty_bytes = bytes_to_human
 
     def human_to_bytes(self, number, isbits=False):
-        return human_to_bytes(number, isbits)
+        return human_to_bytes(number, isbits=isbits)
```

Comment:
do we have an example/test that causes the break?

Reply:
I don't think this code is called anywhere in ansible/ansible. Also codecov doesn't show it being called anywhere in CI: https://app.codecov.io/gh/ansible/ansible/blob/devel/lib%2Fansible%2Fmodule_utils%2Fbasic.py#L2164

Reply:
I added one in 6f78493f33b6dcd6d61e5f7d3b545a02886eff08.